src/mdo/reader-ex.vm

```diff
@@ -57,7 +57,7 @@ import ${packageModelV4}.InputLocation;
 #foreach ( $class in $model.allClasses )
 import ${packageModelV4}.${class.name};
 #end
-import org.codehaus.plexus.util.ReaderFactory;
+import org.codehaus.plexus.util.xml.XmlStreamReader;
```

Comment:
We should avoid direct usage of XmlStreamReader and let the parser to its job, passing either the `InputStream` or `Reader`, but not wrapping the object.